src/components/Navigation.tsx

```diff
@@ -1,109 +1,153 @@
+    '/resources': 'RESOURCES',
+  };
 
-      <Link
-        _hover={{
-          textDecoration: 'underline',
-          textDecorationColor: '#65BF73',
-          textDecorationThickness: '3px',
-        }}
-        fontFamily="Avenir"
-        fontWeight="850"
-        fontSize="16px"
-        as={GatsbyLink}
-        to="/events-and-classes/"
-      >
-        EVENTS & CLASSES
-      </Link>
+  const { isOpen, onOpen, onClose } = useDisclosure();
 
-      <Link
-        _hover={{
-          textDecoration: 'underline',
-          textDecorationColor: '#65BF73',
-          textDecorationThickness: '3px',
-        }}
-        fontFamily="Avenir"
-        fontWeight="850"
-        fontSize="16px"
-        as={GatsbyLink}
-        to="/blog/"
+  return (
+    <>
+      <Flex
+        h="55px"
+        wrap="nowrap"
+        mx="auto"
+        flexDir="row"
+        w="full"
+        justifyContent="space-evenly"
+        alignItems="center"
+        display={['none', null, 'flex']}
       >
-        BLOG
-      </Link>
+        {Object.keys(navigationItems).map(link => {
+          return <NavLink link={link} text={navigationItems[link]} key={link} />;
+        })}
+      </Flex>
 
-      <Link
-        _hover={{
-          textDecoration: 'underline',
-          textDecorationColor: '#65BF73',
-          textDecorationThickness: '3px',
-        }}
-        fontFamily="Avenir"
-        fontWeight="850"
-        fontSize="16px"
-        as={GatsbyLink}
-        to="/resources/"
+      <Flex
+        wrap="nowrap"
+        mx="auto"
+        flexDir="row"
+        w="full"
+        alignItems="center"
+        display={['flex', null, 'none']}
+        px={6}
+        my={3}
+        borderTop="1px"
+        borderTopColor="gray.400"
       >
-        RESOURCES
-      </Link>
-    </Flex>
+        <Drawer placement="left" onClose={onClose} isOpen={isOpen} size="xs">
+          <DrawerOverlay>
+            <DrawerContent background="#404040" color="white">
+              <DrawerHeader>
+                <CloseIcon
+                  w={4}
+                  h={4}
+                  float="right"
+                  onClick={onClose}
+                  _hover={{ cursor: 'pointer' }}
+                />
+              </DrawerHeader>
+              <DrawerBody>
+                <Flex direction="column">
+                  {Object.keys(navigationItems).map(link => {
+                    return (
+                      <MobileNavLink
+                        link={link}
+                        text={navigationItems[link]}
+                        key={link}
+                        onClick={onClose}
+                      />
+                    );
+                  })}
+                </Flex>
+              </DrawerBody>
+            </DrawerContent>
+          </DrawerOverlay>
+        </Drawer>
+
+        <HamburgerIcon h={6} w={6} color="gray.700" onClick={onOpen} />
+        <Box w="178px" display={['block', null, 'none']} margin="auto">
```

Comment:
I don't love the idea of having the image in `Navigation` in addition to `Header`, but c'est la vie

Reply:
Updated the layout to move the HFHF Image to the header on mobile view.